web/themes/custom/lex/templates/views/views-view--taxonomy-term--page-1.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends 'views-view.html.twig' %}
+
+{% block rows %}
+  {% set view = drupal_view_result('taxonomy_term', 'block_term_external_links')|length %}
+  {% if view > 0 %}
+    {{ drupal_view('taxonomy_term', 'block_term_external_links') }}
```

Comment:
I think we could do this with an attachment and let it live in config without having to store it here in a twig. Have you tried that?

Reply:
I haven't tried that, let me give that a look